app/decorators/physical_server_decorator.rb

```diff
@@ -0,0 +1,14 @@
+class PhysicalServerDecorator < MiqDecorator
+  def img_for_physical_vendor
+    "svg/vendor-#{label_for_vendor.downcase}.svg"
+  end
+
+  def img_for_health_state
```

Comment:
Yeah, this is weird. In fact I'd just keep it where it was, since we have no standard way of doing this in decorators yet.

I mean, decorators are really useful if you want to get an icon for *something* you know is a model, but not sure which one.

Any use of the decorators for a method that is not available in any other decorator is ..a bit suspect..

(I mean, what's the point of doing `@record.decorate.img_for_health_state` if you know the type of the record and can't ever call it on any other kind.)

But.. this is a tech debt either way, so.. not saying you have to move it out, just that there is no point in putting it there :).